onward/app/views/mostPopular.scala.html

```diff
@@ -1,26 +1,41 @@
 @(page: model.Page, popular: Seq[model.MostPopular])(implicit request: RequestHeader, context: model.ApplicationContext)
 
-@import views.support.`package`._
+@import layout.FaciaCardHeader
+@import model.Pillar.RichPillar
+@import views.support._
+@import views.html.fragments.items.elements.facia_cards.title
+@import TrailCssClasses.toneClass
 
 @mainLegacy(page){
 }{
     <main itemprop="mainContentOfPage" role="main" class="monocolumn-wrapper content">
         <h1 class="u-h">Most viewed</h1>
-    @popular.zipWithRowInfo.map{ case (section, info) =>
-        <section class="zone-@section.section">
-            <h2 class="content__inline-section">Most viewed @Html(section.heading)</h2>
-            <div class="headline-list"
-                 data-link-name="@section.heading">
-                <ul class="u-unstyled">
+        @popular.map{ section =>
+            <section class="zone-@section.section">
+                <h2 class="content__inline-section">Most viewed @Html(section.heading)</h2>
+                <ul class="u-unstyled headline-list headline-list--large" data-link-name="@section.heading">
                     @section.trails.zipWithRowInfo.map{ case (trail, info) =>
-                    <li class="headline-list__item">
-                        <span class="headline-list__count">@info.rowNum</span>
-                        <div class="headline-list__body">@fragments.linkText(trail, info)</div>
-                    </li>
+                        <li class="headline-list__item @toneClass(trail)--most-popular fc-item--pillar-@trail.maybePillar.nameOrDefault">
```

Comment:
This is exactly the same code, modulo column classes, than in popular.scala.html

Worth factoring that part out in a refactoring